src/mount.ts

```diff
@@ -245,8 +245,40 @@ export function mount(
   return createWrapper(app, App, setProps)
 }
 
+// Functional component
+export function shallowMount<TestedComponent extends FunctionalComponent>(
```

Comment:
If these are the same can we not share the type somehow? Or is this not possible? (Eg, are we not copy-pasting the same 50 or so lines twice?)

Reply:
This is a good question, but I don't know how we could easily do that. @pikax maybe?

Reply:
is a bit tricky and dirty, but we could have something like this: 

```ts
export const shallowMount: typeof mount = (...args) => {
  mount(args[0], { ...args[1], shallow: true })
}
```
Typescript complains about it, but we can cast the function to `unknown` first

Reply:
@pikax I pushed an extra commit with your suggestion 👍 TSD is happy, so it's good enough for me :)